docs/src/main/asciidoc/hibernate-orm-panache-guide.adoc

```diff
@@ -188,6 +188,9 @@ List<String> namesButEmmanuels = persons
     .collect(Collectors.toList());
 --
 
+NOTE: The `stream` methods requires a transaction to work properly.
```

Comment:
Change "requires" to "require"

Reply:
Maybe also drop "properly"? Either it works or it doesn't

Reply:
done